config/migrations/20210512164047-climate-subsidy/20210512164047-replace-ext-namespaces.sparql

```diff
@@ -0,0 +1,55 @@
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX sbc: <http://data.lblod.info/vocabularies/subsidie/climate/>
+PREFIX dummy: <http://data.lblod.info/id/climate/dummy/for/>
+
+# Avoid sprintf error
+INSERT DATA {
+  GRAPH <http://dummy/data/graph/0dc61f81-6ec6-4e16-80d3-26a61079c3c5> {
+    dummy:sprintf a <http://data.lblod.info/dummy/Type> .
+  }
+}
+
+DELETE {
+  GRAPH ?g {
+    dummy:sprintf a ?dummyType.
+    ?sub ext:actionDescription ?description;
+          ext:toRealiseUnits ?toRealiseUnitsValue;
+          ext:costPerUnit ?costPerUnitValue;
+          ext:restitution ?restitutionValue;
+          ext:amountPerAction ?amountPerActionValue.
+
+  }
+} 
+INSERT {
+  GRAPH ?g {
+    ext:sprintf a ?dummyType.
+    ?sub sbc:actionDescription ?description;
+          sbc:toRealiseUnits ?toRealiseUnitsValue;
+          sbc:costPerUnit ?costPerUnitValue;
+          sbc:restitution ?restitutionValue;
+          sbc:amountPerAction ?amountPerActionValue.
+    }
+} 
+WHERE {
+  GRAPH ?g {
+    dummy:sprintf a ?dummyType.
+    OPTIONAL {
```

Comment:
No this won't work in all cases either. The optional will still be considered as a block. Every statement should be optional.

Reply:
Noted